src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralCompilingTests.cs

```diff
@@ -1248,6 +1248,40 @@ public void MultiLineCase34()
                 Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "  ").WithLocation(5, 1));
     }
 
+    [Fact]
+    public void MultiLineCase34_A()
+    {
+        RenderAndVerify(@"
+System.Console.Write(
+    $""""""
+    a
+{42}
```

Comment:
Do we already have a test that shows the expression in the interpolation hole doesn't need to have the same leading whitespace?

```cs
var s = """
    hello {
1 + 2 }
    """;

var s = """
    hello
    {
1 + 2
} world
    """;
```

I assume the `{` that begins the hole needs to be properly indented. What about if the line starts with the `}` that closes a hole? Does that also need to be indented? Maybe not, and it's sort of treated as being part of the same line of content as the `{`?

Reply:
> Do we already have a test that shows the expression in the interpolation hole doesn't need to have the same leading whitespace?

Yes.

> What about if the line starts with the } that closes a hole?

That's an interesting question.  IMO, that whitespace is part of the expression.  It's *inside* the interpolation.  As such, it is not under the same rule.